src/code/PreRender.c

```diff
@@ -551,9 +431,6 @@ void PreRender_Calc(PreRender* this) {
 
                 a >>= 5;
                 a++;
-                if (a != 8) {
```

Comment:
If you look closely this isn't really the best way to do this. You removed the part of the loop that is actually doing something, and the rest of this function is now completely useless. You can just never call `PreRender_Calc` and remove this function and any other functions that it calls.